paddle/platform/device_context.h

```diff
@@ -74,8 +75,7 @@ class CUDADeviceContext : public DeviceContext {
  public:
   explicit CUDADeviceContext(const GPUPlace gpu_place) : gpu_place_(gpu_place) {
     GPUPlaceGuard guard(gpu_place_);
-    paddle::platform::throw_on_error(cudaStreamCreate(&stream_),
-                                     "cudaStreamCreate failed");
+    PADDLE_ENFORCE(cudaStreamCreate(&stream_), "cudaStreamCreate failed");
```

Comment:
Agree to @reyoung. I check on how glog does. What ```CHECK``` does is to check the condition is true. 
And we can implement ```CUDNN_ENFORCE/CUBLAS_ENFORCE``` based on ```PADDLE_ENFORCE``` for writing codes more conviently